pkg/kn/commands/service/service_list.go

```diff
@@ -61,6 +62,10 @@ func NewServiceListCommand(p *commands.KnParams) *cobra.Command {
 				return err
 			}
 
+			//sort serviceList by name
```

Comment:
nit: Comments should be capitalized and have one space at the beginning

Reply:
fixed